Source/Urho3D/CSharp/Native/SWIGHelpers.h

```diff
@@ -12,7 +12,18 @@ template<typename T> T& deref(SwigValueWrapper<T>* ptr) { return (T&)*ptr; }
 namespace pod
 {
 
-#define DEFINE_POD_HELPER_STRUCT(type, n) struct type##n { type data[n]; };
+#define DEFINE_POD_HELPER_STRUCT(type, n) \
+    struct type##n \
+    { \
+        type##n() \
+        { \
+        } \
+        type##n(type v) \
+        { \
+            data[0] = v; \
```

Comment:
Constructors make this type non-pod though. It probably still works, but now is UB. A comment detailing why constructors were added and what are implications would be good here.

Reply:
The fix is merged in. Thanks!